_po/ja/how-to/laravel.po

```diff
@@ -437,38 +642,57 @@ msgid ""
 "ofile.destroy');\n"
 "});"
 msgstr ""
+"Route::middleware('auth')->group(function () {\n"
+"    Route::get('/profile', [ProfileController::class, 'edit'])->name('profile."
+"edit');\n"
+"    Route::patch('/profile', [ProfileController::class, 'update'])->name('prof"
+"ile.update');\n"
+"    Route::delete('/profile', [ProfileController::class, 'destroy'])->name('pr"
+"ofile.destroy');\n"
+"});"
 
 msgid ""
 "require __DIR__.'/auth.php';\n"
 "```"
 msgstr ""
+"require __DIR__.'/auth.php';\n"
+"```"
 
 msgid "This will create the following routes:"
-msgstr ""
+msgstr "これにより次のルートが作成されます:"
 
 msgid ""
 "| Verb | URI         | Action  | Route Name     |\n"
 "|------|-------------|---------|----------------|\n"
 "| GET  | `/`         | index   | posts.index    |\n"
 "| GET  | `/search`   | search  | posts.search   |"
 msgstr ""
+"| Verb | URI         | Action  | Route Name     |\n"
+"|------|-------------|---------|----------------|\n"
+"| GET  | `/`         | index   | posts.index    |\n"
+"| GET  | `/search`   | search  | posts.search   |"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.